app/models/event_schedule.rb

```diff
@@ -37,6 +39,20 @@ def replacement?
 
   private
 
+  def start_after_end_hour
+    if event && event.program && event.program.conference && event.program.conference.end_hour && start_time && start_time.strftime('%H').to_i > event.program.conference.end_hour
```

Comment:
This is long and gets confusing, what about we re-write it as:

return unless event && start_time && event.program && event.program.conference && event.program.conference.end_hour

and then have a single line with the 
errors.add() if start_time.strftime('%H').to_i > event.program.conference.end_hour